README.md

```diff
@@ -137,6 +139,12 @@ For ease of use with npm scripts and for quick testing.
     -p, --pod-module-prefix [pod-module-prefix]  Pod Module Prefix, the directroy that the ember-resolver uses for pods
 ```
 
+### Ember Addons
+
+There is basic support for ember-addons. You should be able to npm install the addon, and assuming the published addon conforms to ember's addon standard, emberate will include the addon in your app bundle entry file.
+
+You can optionally exclude the addon support from your build by setting `addonSupport` to false in the emberate options. Since Ember Addons are normally written in es6, you will need to include a transpiler in your browserify bundle. The most simple way to do so is with the [babelify transform](https://github.com/babel/babelify).
+
```

Comment:
Maybe worth noting that default blueprints would have to be run manually, not sure if you've thought of that.